packages/portfolio-contract/ymax-machine.yaml

```diff
@@ -0,0 +1,156 @@
+# yaml-language-server: $schema=./ymax-machine.schema.json
```

Comment:
oh! I was under the impression that the .mmd was the source

I'm much more comfortable maintaining this .yaml file